### DISCUSSION TITLE
Private mode in cameras

### DISCUSSION THREAD

**Discussion by jbrandek** [tsightler/ring-mqtt#741] [Feature Requests]
Any chance to add the option to activate private mode in cameras? So it will be possible to schedule it?

**Comment by tsightler** [comment]
What is private mode?  A search doesn't produce any feature that matches that name.

**Reply by jbrandek** [reply]
Sorry its call disarmed in rhis cameras, it desacrivate everything cannot see the cameras and they do not record

![IMG-20231004-WA0001.jpg](https://github.com/tsightler/ring-mqtt/assets/51308256/82ea65a4-9dcb-412b-97cd-1950bed2607f)

**Reply by tsightler** [reply]
Ring location modes are already supported and have been for many years.  Just set "enable_modes" to true and restart ring-mqtt and you will get an alarm control panel entity that allows setting the camera modes to disarmed/home/away modes.  This is covered in the docs.

**Reply by jbrandek** [reply]
Thanks
